modules/covid_vaccine/app/services/covid_vaccine/v0/enrollment_upload_service.rb

```diff
@@ -34,6 +41,14 @@ def sftp_password
       def sftp_port
         Settings.covid_vaccine.enrollment_service.sftp.port
       end
+
+      def sftp_kex_algorithms
+        Settings.covid_vaccine.enrollment_service.sftp.kex_algorithms.split(',')
+      end
+
+      def sftp_host_key_algorithms
+        Settings.covid_vaccine.enrollment_service.sftp.host_key_algorithms.split(',')
```

Comment:
These don't exist for lower environments. Is that okay? Or does settings.yaml need to be updated with default values?

Reply:
Added in placeholders are this doesn't run in lowers